### PR TITLE
Clean current and future warnings

### PR DESCRIPTION
**Pull request by cepages**
@incuna/appdev please review

### CODE REVIEW

File: INCWaveMountainsView/Classes/INCWaveMountainLayer.h [hunk 1]
```diff
@@ -21,19 +23,37 @@
...
 
-@property(nonatomic,assign)BOOL unblockMontainsForMissingPercents;
-@property(nonatomic,weak,nullable)id<INCWavwMountainLayerDelegate> delegate;
+@property(nonatomic,strong,nonnull)CAShapeLayer *shapeLayerMountain;//This property define the shape of the mountain
+@property(nonatomic,assign)float mountainPercent;//This define the hight of the mountain, the value has to be between 0 to 1 or NO_PERCENT_VALUE. A lower value will be considered 0 and higher value will be considered 1.
+@property(atomic,strong,nullable)NSNumber *mountainId;//This is the id of the mountain
+@property(nonatomic,strong,nonnull)CAShapeLayer *maskMountain;//Mask layer
+@property(nonatomic,strong,nonnull)CAGradientLayer *gradientMountain;//Gradient colour
+
+@property(nonatomic,assign)BOOL unblockMontainsForMissingPercents;//If yes the time can be set up to unblock a mountain automatically. If after unblockTimeInterval time the layer doesn't new values the mountain is reset to 0.
```
Comment (wytrych) [inline INCWaveMountainsView/Classes/INCWaveMountainLayer.h:33]: I don't think these comments are very helpful. If you think the name doesn't explain itself enough, change it :)

File: INCWaveMountainsView/Classes/INCWaveMountainLayer.m [hunk 2]
```diff
@@ -8,6 +8,13 @@
 
 #import "INCWaveMountainLayer.h"
 
+const NSInteger NO_PERCENT_VALUE = -1;
+
```
Comment (wytrych) [inline INCWaveMountainsView/Classes/INCWaveMountainLayer.m]: Remove line?

File: INCWaveMountainsView/Classes/INCWaveMountainLayer.h (hunk 1 above)
Comment (cepages) [reply INCWaveMountainsView/Classes/INCWaveMountainLayer.h:33]: Those comments create an auto documentation in XCode

https://www.raywenderlich.com/66395/documenting-in-xcode-with-headerdoc-tutorial

File: INCWaveMountainsView/Classes/INCWaveMountainLayer.m (hunk 2 above)
Comment (cepages) [reply INCWaveMountainsView/Classes/INCWaveMountainLayer.m]: done

File: INCWaveMountainsView/Classes/INCWaveMountainLayer.h (hunk 1 above)
Comment (cepages) [reply INCWaveMountainsView/Classes/INCWaveMountainLayer.h:33]: Actually let me fix something to see the documentation in the properties

**Pull request merged**